src/button/template/content.json

```diff
@@ -1220,7 +1220,7 @@
             "later_tag": "Jetzt bei uns bequem in Raten zahlen.",
             "pay": "Mit {logo:paypal} zahlen",
             "buynow": "{logo:pp} {logo:paypal} Jetzt kaufen",
-            "poweredBy": "Abgewickelt durch {logo:paypal}"
+            "poweredBy": "Powered by PayPal"
```

Comment:
Not clear on why we're putting english in the DE translations?

Reply:
Also either way we should keep `{logo:paypal}`

Reply:
I had the same Question but in the ticket `PPPLMER-59482` product wanted english in 'DE' translation ( I know .. I know...)

Reply:
I updated the code Daniel, could you check when you get time?